src/bitcoin-cli.cpp

```diff
@@ -228,6 +228,7 @@ class GetinfoRequestHandler: public BaseRequestHandler
     const int ID_NETWORKINFO = 0;
     const int ID_BLOCKCHAININFO = 1;
     const int ID_WALLETINFO = 2;
+    const int ID_BALANCESINFO = 3;
```

Comment:
nit: following the convention that `ID_foo` maps to `getfoo` this should be named `ID_BALANCES`.

Reply:
done